src/Algebra/Properties/BooleanAlgebra.agda

```diff
@@ -26,8 +26,7 @@ open BooleanAlgebra B
 
 import Algebra.Properties.DistributiveLattice as DistribLatticeProperties
 
-open import Algebra.Structures _≈_
-open import Relation.Binary
+open import Relation.Binary using (Rel)
```

Comment:
I think `Rel` is defined in a much smaller module than `Relation.Binary`. It would be worth narrowing this down.

Reply:
As below, though: these are commits to a DEPRECATED module, and as such, I think should be avoided unless absolutely necessary?